### PR TITLE
New bolt authenticator for the City of Santa Monica

### PR DESCRIPTION
**Pull request by RMK0110**
This is a code block for the new Bolt authenticator used in a DAG (dockless_elt.py) at the City of Los Angeles, and was sent to the City of Santa Monica.  CC @hunterowens @ian-r-rose

### CODE REVIEW

**Comment by ian-r-rose** [comment]
This is currently missing some imports (at least `requests` and `AuthorizationToken`). I think the most appropriate place for this would be in `mds/api/auth.py`, next to `SpinAuthorizationToken`. If you put it there, the needed imports should already be in scope.

File: mds/api/auth.py [hunk 1]
```diff
@@ -1,8 +1,11 @@
 """
 Authentication module for MDS API calls.
 """
-
+from mds.api.auth import AuthorizationToken
```
Comment (ian-r-rose) [inline mds/api/auth.py]: This is from this very file, so doesn't need to be imported.

File: mds/api/auth.py [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+import mds
+import mds.providers
```
Comment (ian-r-rose) [inline mds/api/auth.py]: These imports, as with `json` below, are unused.

Comment (RMK0110) [reply mds/api/auth.py]: It is corrected.

File: mds/api/auth.py (hunk 1 above)
Comment (RMK0110) [reply mds/api/auth.py]: It has been corrected.

File: mds/bolt_auth.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+class BoltClientCredentials(AuthorizationToken):
```
Comment (ian-r-rose) [inline mds/bolt_auth.py]: This file should be removed, since it's now in `mds/api/auth.py`.

Comment (RMK0110) [reply mds/bolt_auth.py]: This should be removed now.  - RMK0110

Comment (thekaveman) [reply mds/bolt_auth.py]: Looks like the file is still in the branch @RMK0110 - please delete

File: mds/api/auth.py [hunk 4]
```diff
@@ -1,7 +1,6 @@
 """
 Authentication module for MDS API calls.
 """
-
```
Comment (thekaveman) [inline mds/api/auth.py]: Minor thing, but stylistically we should have a blank line between the docstring and the first import statement.

**Comment by RMK0110** [comment]
The stylistic request should be completed, but let me know.  - RMK0110

**Comment by thekaveman** [comment]
@RMK0110 I don't see any new commits, did you push?

**Comment by ian-r-rose** [comment]
Thanks for the review and merge @thekaveman

**Pull request merged**